Userland/Libraries/LibGfx/Matrix.h

```diff
@@ -84,11 +85,9 @@ class Matrix {
     constexpr Matrix operator/(T divisor) const
     {
         Matrix division;
-        for (size_t i = 0; i < N; ++i) {
-            for (size_t j = 0; j < N; ++j) {
+        for (size_t i = 0; i < N; ++i)
+            for (size_t j = 0; j < N; ++j)
```

Comment:
Please put at least the outer curly braces back. Any if/else/while/for/... body with more than one line should have them.

Reply:
Right - was a bit too eager there!